generation/WinSDK/enums.json

```diff
@@ -31932,5 +31957,49 @@
     },
     "members": [],
     "uses": []
+  },
+  {
+    "name": "EMPTY_VOLUME_CACHE_FLAGS",
+    "namepace": "Windows.Win32.UI.LegacyWindowsEnvironmentFeatures",
```

Comment:
You've got a typo in `namespace` here and it's not necessary anyway since this will inherit the same namespace from IEmptyVolume*. Remove the namespace property here.